dash-pipeline/bmv2/underlay.p4

```diff
@@ -68,8 +68,8 @@ control underlay(
 #endif // TARGET_DPDK_PNA
     }
 
-    @name("route|route")
-    // TODO: To add structural annotations (example: @SaiTable[skipHeaderGen=true])
+    @SaiTable[name = "route", api = "route"]
+    // TODO: To add structural annotations (example: @SaiTable[skipHeaderGen=true]]
```

Comment:
I think this comment can be deleted. The idea of a new attribute `skipHeaderGen` was not followed, instead the api name omits the `dash` which is kind of a heuristic in the api_gen code, see here: https://github.com/sonic-net/DASH/blob/e6a8e98a5de7b7f618ca37c8b10f55c502e5069c/dash-pipeline/SAI/sai_api_gen.py#L910. The purpose was to use existing underlay SAI headers, not try to generate new ones. It was convenient to look for `dash` in the table name but IMO it's still a hidden assumption.

Do you think it'd be more consistent to use the attribute `skipHeaderGen` or some other aptly-named one to explicitly direct the codegen *not* to emit header code for a table? It would default to false.

Reply:
of course! will update the PR soon. (will be afk for a hour or so.)

Reply:
thanks a lot for capturing this! i totally missed this one.

Reply:
Since essentially, the dash prefix affects the generation of both the header files and implementation files, instead of using "skiopHeaderGen", I added a new attribute called "api_type" and set to "underlay" to more precisely reflect what we are doing.

The changes are commited now. Please let me know what you think! :D

Reply:
and of course, no SAI header file change and only table id updates in lib files.

Reply:
ok, test passed now. @chrispsommers , it should be good to table another look now.